src/core/parser/csv_parser/csv_parser.cpp

```diff
@@ -94,6 +103,16 @@ void CSVParser::GetLine(unsigned long long const line_index) {
 
 void CSVParser::GetNextIfHas() {
     has_next_ = !source_.eof();
+
+    /* Check for the last newline */
```

Comment:
```suggestion
    // Check for the last newline
```
Be consistent in comment style